backend/src/db/migrations/20230912094401_delete_name_column_from_chat_messages_table.ts

```diff
@@ -0,0 +1,21 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'chat_messages';
+
+const ColumnName = {
+  NAME: 'name',
+} as const;
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TABLE_NAME, (table) => {
+    table.dropColumn(ColumnName.NAME);
+  });
+}
+
+function down(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TABLE_NAME, (table) => {
+    table.string(ColumnName.NAME).unique().notNullable();
```

Comment:
take a look at the `20230914101723_remove_unique_constraint_in_chats_name_column.ts` migration, we should do something similar - add default name by index like - `Message #1`